src/Tasks.UnitTests/AssemblyDependency/SuggestedRedirects.cs

```diff
@@ -64,7 +64,7 @@ public void ConflictBetweenNonCopyLocalDependencies()
             Assert.True(ContainsItem(t.ResolvedDependencyFiles, s_myLibraries_V2_GDllPath));
 
             Assert.Single(t.SuggestedRedirects);
-            Assert.True(ContainsItem(t.SuggestedRedirects, @"D, Culture=neutral, PublicKeyToken=aaaaaaaaaaaaaaaa")); // "Expected to find suggested redirect, but didn't"
+            Assert.True(ContainsItem(t.SuggestedRedirects, @$"{"D"}, Culture=neutral, PublicKeyToken=aaaaaaaaaaaaaaaa")); // "Expected to find suggested redirect, but didn't"
```

Comment:
??

Reply:
I can undo this one. An earlier version required a legitimate test change but no more.